### ISSUE TITLE
Time pseudotype instead of ISO8601 when using group operator

### ISSUE THREAD

**Issue by marcbc** [neumino/rethinkdbdash#198]
I run across some behaviour when using rethinkdbdash that I suspect is not intended. 

While querying a table that has a datetime attribute I get the iso8601 formatted value for it unless I add a group operator to the query, in which case I'm getting a datetime pseudotype. Doing an ungroup afterwards gives me the iso8601 again.

So I think there is some inconsistency in the way datetimes are formatted, since I would expect them to be always the same way, either iso8601 or datetime.

Eg.
```db.table('apples').max('timestamp');``` returns timestamps as 
```
{
  ...
  "timestamp":"2015-02-04T15:45:02.311Z"
}
```

while ```db.table('apples').group('type').max('timestamp');``` returns them as:
```
{
  ...
  "timestamp": {
    "$reql_type$": "TIME",
    "epoch_time": 1423077646.772,
    "timezone": "-07:00"
  }
}
```

and ```db.table('apples').group('type').max('timestamp').ungroup();``` does return again:
```
{
  ...
  "timestamp":"2015-02-04T15:45:02.311Z"
}
```

**Comment by neumino** [comment]
What version are  you using? There was a fix for that in 2.2.7.

**Comment by marcbc** [comment]
Sorry @neumino , I was using 2.1.x, I upgraded to 2.2.8 and it works fine now. Forget about it. Thanks.
